tools/toolchains/gcc.py

```diff
@@ -36,7 +36,7 @@ class GCC(mbedToolchain):
     STD_LIB_NAME = "lib%s.a"
     DIAGNOSTIC_PATTERN = re.compile('((?P<file>[^:]+):(?P<line>\d+):)(?P<col>\d+):? (?P<severity>warning|[eE]rror|fatal error): (?P<message>.+)')
 
-    GCC_RANGE = (LooseVersion("9.0.0"), LooseVersion("10.0.0"))
+    GCC_RANGE = (LooseVersion("9.0.0"), LooseVersion("11.0.0"))
```

Comment:
the first should be also fixed to 10 (previously it was set to >9, <10, it should be now >10,<11